.travis.yml

```diff
@@ -15,7 +15,7 @@ deploy:
   on:
     tags: true
     condition: "$TRAVIS_PYTHON_VERSION == 3.6"
-  distributions: "sdist bdist_wheel"
+  before_deploy: python translate_templates.py
```

Comment:
have you tried a deploy to test.pypi.org?

Reply:
I will do one manually, but I haven't tried from travis. Unfortunately travis explicitly doesn't run the deploy stage from a PR, so we can't automate a test deployment to test.pypi.org for PRs.

Reply:
Here is the test.pypi.org upload: https://test.pypi.org/project/sphinx-bluebrain-theme/0.1.4.post1/